app/helpers/application_helper.rb

```diff
@@ -85,6 +85,12 @@ def fraction_to_percentage_display(fraction, options = {precision: 2, significan
     end
   end
 
+  def formatted_month(month, include_nbsp = false)
+    return nil unless month
```

Comment:
Explicit returns made this smell to me and now I look at it, why is this check needed at all?